src/main/java/seedu/address/ui/ReminderListPanel.java

```diff
@@ -45,4 +52,28 @@ protected void updateItem(Item item, boolean empty) {
         }
     }
 
+    /**
+     * Scrolls up.
+     */
+    public void scrollUp() {
+        if (currentPosition - NUM_OF_ITEMS_TO_SCROLL <= 0) {
+            currentPosition = 0;
+        } else {
+            currentPosition = currentPosition - NUM_OF_ITEMS_TO_SCROLL;
+        }
+        reminderListView.scrollTo(currentPosition);
+
+    }
+
+    /**
+     * Scrolls down.
+     */
+    public void scrollDown() {
+        if (currentPosition + NUM_OF_ITEMS_TO_SCROLL >= itemSize) {
+            currentPosition = itemSize - NUM_OF_ITEMS_TO_SCROLL;
+        } else {
+            currentPosition = currentPosition + NUM_OF_ITEMS_TO_SCROLL;
+        }
+        reminderListView.scrollTo(currentPosition);
+    }
```

Comment:
Since this two methods are repeated across all the panels, would you consider adding this to an abstract class and then extending from that abstract class?

Reply:
I will do this in the next iteration.